src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
+    public static final int MAX_NUMBER_BALL = 101;
```

Comment:
ball model shouldn't be concerned with issues of Lottery implementation like this - move it to Lottery class, and there you can make it private